packages/paste-core/components/badge/stories/index.stories.tsx

```diff
@@ -26,7 +26,7 @@ const Wrapper = styled.div(
   })
 );
 
-export const AllBadges: React.FC = () => (
+export const AllBadges = (): JSX.Element => (
```

Comment:
suggest (non-blocking): If you were going to the effort of swapping all these, you _could_ swap them to 

```
export const AllBadges:StoryFn = () => (
```

Which returns a JSX.Element